components/tag/stories/tag.test.js

```diff
@@ -1,51 +1,102 @@
 import { Variants } from "@spectrum-css/preview/decorators";
-import { Template } from "./template.js";
+import { TagsDefaultOptions, Template } from "./template.js";
 
 export const TagGroups = Variants({
 	Template,
+	SizeTemplate: TagsDefaultOptions,
 	sizeDirection: "row",
 	testData: [
 		{
 			testHeading: "Default",
 		},
 		{
-			testHeading: "Is removable",
+			testHeading: "Emphasized",
+			isEmphasized: true,
+		},
+		{
+			testHeading: "Selected",
+			isSelected: true,
+		},
+		{
+			testHeading: "Emphasized, selected",
+			isEmphasized: true,
+			isSelected: true,
+		},
+		// show removable tags for these variants so hover and focus can be captured
+		{
+			testHeading: "Default, removable",
+			hasClearButton: true,
+		},
+		{
+			testHeading: "Emphasized, removable",
+			isEmphasized: true,
 			hasClearButton: true,
 		},
 		{
-			testHeading: "With icon",
-			hasIcon: true,
-			iconName: "Info",
+			testHeading: "Selected, removable",
+			isSelected: true,
+			hasClearButton: true,
+		},
+		{
+			testHeading: "Disabled with states",
+			isDisabled: true,
+		},
+		// variants with visuals
+		{
+			testHeading: "Default, with icon",
+			iconName: "Circle",
 		},
 		{
-			testHeading: "With avatar",
-			hasAvatar: true,
+			testHeading: "Default, with avatar",
 			avatarUrl: "example-ava.png",
 		},
+		{
+			testHeading: "Emphasized, with thumbnail",
+			isEmphasized: true,
+			thumbnailUrl: "example-card-landscape.png",
+		},
+		// read-only
+		{
+			testHeading: "Default, read-only",
+			isReadOnly: true,
+		},
+		{
+			testHeading: "Emphasized, read-only",
+			isReadOnly: true,
+			isEmphasized: true,
+		},
```

Comment:
This one was showing emphasized colors but I would expect it to show read-only colors.